packages/halo-theme/src/custom-elements/ef-toggle.less

```diff
@@ -54,10 +54,10 @@
     &:hover:not([readonly]) {
       color: @input-hover-text-color;
       [part=toggle] {
-        border-color: @scheme-color-primary;
+        border-color: @toggle-switch-background-hover-border-color;
         &::after {
-          background-color: @scheme-color-primary;
-          border-color: @scheme-color-primary;
+          background-color: @toggle-switch-background-hover-color;
+          border-color: @toggle-switch-background-hover-border-color;
```

Comment:
I think we can reuse @button-hover-background-color and @button-hover-border-color instead.

Reply:
Oh, I got it now. thanks for mentioning this. updated